game/scripts/vscripts/heroes/vengefulspirit/command_aura.lua

```diff
@@ -0,0 +1,204 @@
+LinkLuaModifier("modifier_vengefulspirit_command_aura_effect_lua", "heroes/vengefulspirit/command_aura", LUA_MODIFIER_MOTION_NONE)
+LinkLuaModifier("modifier_vengefulspirit_command_aura_lua", "heroes/vengefulspirit/command_aura", LUA_MODIFIER_MOTION_NONE)
+
+--[[
+==============================
+===== Command Aura Effect ====
+==============================
+--]]
+---@class modifier_vengefulspirit_command_aura_effect_lua : CDOTA_Modifier_Lua
+---@field bonus_damage_pct number
+modifier_vengefulspirit_command_aura_effect_lua = class({})
+
+---@override
+---@return boolean
```

Comment:
return boolean is on the doc of the overridden spell, not needed here, same for all other ---@override functions, you only need `---@override`

```
---@override
function modifier_vengefulspirit_command_aura_effect_lua:IsDebuff()
    ...
```